env_manager/tests/test_manager.py

```diff
@@ -3,105 +3,194 @@
+    BACKENDS,
+    indirect=["manager_instance"],
+)
+def test_manager_backends(
+    manager_instance, initial_package, installed_packages, errored_packages, messages
+):
     # Create an environment with Python in it
-    manager.create_environment(packages=["python"])
-    initial_list = manager.list()
+    manager_instance.create_environment(packages=["python"])
+    initial_list = manager_instance.list()
     assert initial_package in " ".join(initial_list)
 
     # Install a new package in the created environment
-    install_result = manager.install(packages=installed_packages, force=True)
+    install_result = manager_instance.install(packages=installed_packages, force=True)
     assert install_result[0]
 
-    def package_installed():
-        package_list = manager.list()
-        for package in installed_packages:
-            return package in " ".join(package_list)
-
-    wait_until(package_installed)
+    # Check package was installed
+    wait_until(
+        package_installed,
+        manager_instance=manager_instance,
+        installed_packages=installed_packages,
+    )
 
     # Uninstall the new package
-    uninstall_result = manager.uninstall(packages=installed_packages, force=True)
+    uninstall_result = manager_instance.uninstall(
+        packages=installed_packages, force=True
+    )
     assert uninstall_result[0]
 
-    def packages_uninstalled():
-        package_list = manager.list()
-        for package in installed_packages:
-            return package not in " ".join(package_list)
-
-    wait_until(packages_uninstalled)
+    # Check package was uninstalled
+    wait_until(
+        packages_uninstalled,
+        manager_instance=manager_instance,
+        installed_packages=installed_packages,
+    )
 
     # Try to install unexisting package
     install_error_message, uninstall_warning_message = messages
-    error_result, error_message = manager.install(packages=errored_packages, force=True)
+    error_result, error_message = manager_instance.install(
+        packages=errored_packages, force=True
+    )
     assert not error_result
     assert install_error_message in error_message
 
     # Try to uninstall unexisting package
-    warning_result, subprocess_result = manager.uninstall(
+    warning_result, subprocess_result = manager_instance.uninstall(
         packages=errored_packages, force=True
     )
     assert warning_result
     assert (
         uninstall_warning_message in subprocess_result.stdout + subprocess_result.stderr
     )
+
+
+@pytest.mark.parametrize(
+    "manager_instance,initial_import_path,expected_export_path",
+    IMPORT_EXPORT_BACKENDS,
+    indirect=["manager_instance"],
+)
+def test_manager_backends_import_export(
+    manager_instance, initial_import_path, expected_export_path, tmp_path
+):
+    # Import environment definition
+    import_result = manager_instance.import_environment(initial_import_path)
+    print(import_result[1])
```

Comment:
Is this `print` necessary?

Reply:
I left it there since if the test fails for some reason this enables to check the reason the command failed. Should I remove it?

Reply:
No, I think it's fine to leave. I just wasn't sure if it was a left over or really necessary.